libraries/AP_Logger/LogFile.cpp

```diff
@@ -188,23 +188,25 @@ void AP_Logger::Write_RCOUT(void)
     const uint32_t enabled_mask = ~SRV_Channels::get_output_channel_mask(SRV_Channel::k_GPIO);
 
     if ((enabled_mask & 0x3FFF) != 0) {
+        uint16_t channels[14] {};
```

Comment:
Could you use `NUM_SERVO_CHANNELS` here and use this method for the full 32?

Reply:
> Could you use `NUM_SERVO_CHANNELS` here and use this method for the full 32?

I did consider that - but I figured I might be causing unnecessary work when doing the fetching.

Essentially I really just decided to take the easy win - definitely not saying we can't do better in here.